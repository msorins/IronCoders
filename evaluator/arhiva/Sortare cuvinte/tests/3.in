yoy
yijy
sadjkoasdja
asdjkasdbji
a
sda
a
a
sfafoafj
nn
ashodaid
aa
zvv
v
saaa
afa
vgkorgojo
