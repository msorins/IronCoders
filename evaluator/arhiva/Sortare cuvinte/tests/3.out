a
a
a
aa
afa
asdjkasdbji
ashodaid
nn
saaa
sadjkoasdja
sda
sfafoafj
v
vgkorgojo
yijy
yoy
zvv
